keyboards/kbdfans/kbd6x/keymaps/via/keymap.c

```diff
@@ -0,0 +1,50 @@
+/* Copyright 2018 MechMerlin
```

Comment:
You can put your name here!

Reply:
Done and done! Thanks, Merlin! 

Do you have any suggestions for how to resolve the above LTO issue? My board is currently flashed with @drashna 's solution and working fine. But, I also understand the simplicity of @fauxpark's suggestion (which also works with the core change applied).